source/hassio/flashing_n2_otg.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "Flashing an ODROID-N2+"
+description: "Using Petitboot and OTG-USB to flash the eMMC on your Odroid N2"
+---
+
+Home Assistant can be flashed to an ODROID-N2+ by connecting the device directly to your computer via the USB-OTG connection on the front of the board. The device contains the Petitboot bootloader which allows the ODROID-N2+ storage to show up as it were a USB drive.
+
+_All these instructions work the same for the ODROID-N2 (non-plus version)._
+
+## What you will need
+
+In order to flash your eMMC using Petitboot and OTG-USB you will need the following items:
+
+- HDMI cable and monitor
+- USB keyboard
+- USB 2.0 to micro-usb cable
+
+---
+
+### Enabling SPI boot mode
+
+Remove the case of your ODROID-N2+
+
+<img src='/images/hassio/screenshots/cased-removed.jpg' />
+
+Next, locate the toggle for boot mode and switch it from MMC to SPI.
+
+<img src='/images/hassio/screenshots/toggle_spi.png' />
+
+Connect a USB keyboard and HDMI connected monitor to your ODROID-N2+, and then connect power.
+
+### Enabling USB drive mode
+
+The ODROID-N2+ will now boot into a terminal. Select `Exit to shell` from the menu.
+
+<img src='/images/hassio/screenshots/exit-shell.png' />
+
+Use the following command at the console to confirm the storage device node:
+
+```bash
+ls /dev/mmc*
+```
+
+Set the storage device on the ODROID-N2+ as a mass storage device using `ums` (USB Mass storage mode)
+This will configure the ODROID-N2+ and OTG to act as a memory card reader.
+
+```bash
+ums /dev/mmcblk0
```

Comment:
It's not clear here that we tell people to use the path that they found above. 

I wonder, can we always set it to `mmcblk0` ? I don't think I ever saw it differently.

Reply:
The above is fine for the general setup.

Reply:
I believe it will always be mmcblk0 for our purposes? I never saw anything else either.